### ISSUE TITLE
Unhandled exception in RatScanner 3.0.0

### ISSUE THREAD

**Issue by shhwang7** [RatScanner/RatScanner#148]
**Error**
Unhandled exception in RatScanner 3.0.0
```
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> Microsoft.Web.WebView2.Core.WebView2RuntimeNotFoundException: Couldn't find a compatible Webview2 Runtime installation to host WebViews.
 ---> System.IO.FileNotFoundException: 지정된 파일을 찾을 수 없습니다. (0x80070002)
   --- End of inner exception stack trace ---
   at Microsoft.Web.WebView2.Core.CoreWebView2Environment.CreateAsync(String browserExecutableFolder, String userDataFolder, CoreWebView2EnvironmentOptions options)
   at Microsoft.AspNetCore.Components.WebView.Wpf.WpfWebView2Wrapper.CreateEnvironmentAsync()
   at Microsoft.AspNetCore.Components.WebView.WebView2.WebView2WebViewManager.InitializeWebView2()
   at Microsoft.AspNetCore.Components.WebView.WebView2.WebView2WebViewManager.<>c__DisplayClass4_0.<<NavigateCore>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Components.WebView.Wpf.WpfDispatcher.InvokeAsync(Func`1 workI
```
<details>
<summary>Log</summary>

```
[04:30:00.1859203] > [Error] Unhandled exception in RatScanner 3.0.0
 -------------------- 
 System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> Microsoft.Web.WebView2.Core.WebView2RuntimeNotFoundException: Couldn't find a compatible Webview2 Runtime installation to host WebViews.
 ---> System.IO.FileNotFoundException: 지정된 파일을 찾을 수 없습니다. (0x80070002)
   --- End of inner exception stack trace ---
   at Microsoft.Web.WebView2.Core.CoreWebView2Environment.CreateAsync(String browserExecutableFolder, String userDataFolder, CoreWebView2EnvironmentOptions options)
   at Microsoft.AspNetCore.Components.WebView.Wpf.WpfWebView2Wrapper.CreateEnvironmentAsync()
   at Microsoft.AspNetCore.Components.WebView.WebView2.WebView2WebViewManager.InitializeWebView2()
   at Microsoft.AspNetCore.Components.WebView.WebView2.WebView2WebViewManager.<>c__DisplayClass4_0.<<NavigateCore>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Components.WebView.Wpf.WpfDispatcher.InvokeAsync(Func`1 workItem)
   at Microsoft.AspNetCore.Components.WebView.Wpf.WpfDispatcher.<>c.<.cctor>b__8_0(Exception exception)
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Span`1& arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Delegate.DynamicInvokeImpl(Object[] args)
   at System.Delegate.DynamicInvoke(Object[] args)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
[04:30:00.4822273] > [Error] Unhandled exception in RatScanner 3.0.0
 -------------------- 
 System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> Microsoft.Web.WebView2.Core.WebView2RuntimeNotFoundException: Couldn't find a compatible Webview2 Runtime installation to host WebViews.
 ---> System.IO.FileNotFoundException: 지정된 파일을 찾을 수 없습니다. (0x80070002)
   --- End of inner exception stack trace ---
   at Microsoft.Web.WebView2.Core.CoreWebView2Environment.CreateAsync(String browserExecutableFolder, String userDataFolder, CoreWebView2EnvironmentOptions options)
   at Microsoft.AspNetCore.Components.WebView.Wpf.WpfWebView2Wrapper.CreateEnvironmentAsync()
   at Microsoft.AspNetCore.Components.WebView.WebView2.WebView2WebViewManager.InitializeWebView2()
   at Microsoft.AspNetCore.Components.WebView.WebView2.WebView2WebViewManager.<>c__DisplayClass4_0.<<NavigateCore>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Components.WebView.Wpf.WpfDispatcher.InvokeAsync(Func`1 workItem)
```
</details>

**Comment by jaybocc2** [comment]
fix is to download and install webview2 https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section

**Comment by 2m** [comment]
> fix is to download and install webview2 [developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section](https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section)

Thanks! This helped me as well!

**Comment by Blightbuster** [comment]
Fixed in [v3.1.0](https://github.com/RatScanner/RatScanner/releases/tag/v3.1.0).
You no longer need to install any dependencies.
Please comment if this issue persists and I will reopen it.
